templates/cinder/config/00-global-defaults.conf

```diff
@@ -45,9 +45,11 @@ lock_path = /var/locks/openstack/os-brick
 [oslo_concurrency]
 lock_path = /var/locks/openstack/cinder
 
-# TODO:
+# Notifications are disabled by default, they can be easily enabled by a human
```

Comment:
FWIW, tripleo configures this with driver=noop. We could do the same here, and ask the operator to just override the driver. Configuring the transport_url here isn't a huge help to the operator because "{{ .TransportURL }}" makes it easy for them to configure it.
That said, if we intend to have the operator configure both the driver and transport_url in their customServiceConfig, then do we need to leave these comments in the global defaults file at all?

Reply:
Even if setting the `transport_url` is not a great help, I do think it's convenient, so I'll change this to a `noop`.
And writing this comment here was to make sure we had it documented in our repository that disabling it was on purpose and how to configure it in a convenient way.